modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -4861,27 +4861,25 @@ private void checkConnection() {
                     else
                         srvrSock = new ServerSocket(port, 0, spi.locHost);
 
-                    break;
+                    if (log.isInfoEnabled())
```

Comment:
Let's do the following here

Create 'lastPort' variable
int lastPort = spi.locPortRange  == 0 ? spi.locPort : spi.locPort + spi.locPortRange - 1;

for (port = spi.locPort; port <= lasPort; port++)
....